CRM/Event/Page/ManageEvent.php

```diff
@@ -208,10 +198,6 @@ public static function &tabs() {
         ];
     }
 
-    if (!$enableCart) {
-      unset(self::$_tabLinks[$cacheKey]['conference']);
-    }
```

Comment:
Funny how CiviCRM was modeled after Drupal, which uses hooks for literally everything, yet we ended up with code like this.